src/init.cpp

```diff
@@ -195,6 +195,7 @@ void DynamicDataLoader::initialize()
     add( "profession_item_substitutions", &profession::load_item_substitutions );
     add( "skill", &Skill::load_skill );
     add( "skill_display_type", &SkillDisplayType::load );
+    add( "bionics_display_type", &BionicsDisplayType::load );
```

Comment:
This is loaded, but never reset (e.g. when the player saves the current game and loads a new one, possible with different display types).